main/res/values-ca/strings.xml

```diff
@@ -92,7 +92,7 @@
   <string name="log_posting_twitter">Publicant Tweet…</string>
   <string name="log_posting_gcvote">Publicant GCvote…</string>
   <string name="log_posting_image">Publicant imatge…</string>
-  <string name="log_posting_generic_trackable">Publicant %1$s%2$s/%3$s</string>
+  <string name="log_posting_generic_trackable">Publicant %1$s%2$d/%3$d</string>
```

Comment:
Please do not modify strings outside the source (`res/values/strings.xml`). Other ones come from Crowdin.